src/utils/tree/getNodeTags/getNodeTags.ts

```diff
@@ -0,0 +1,30 @@
+import type { TreeNode } from "@/features/Treege/type/TreeNode";
+
+/**
+ * Get node names from tree
```

Comment:
```suggestion
 * Get node tags from tree
```

PS: Toutes les opti proposé pour ce fichier peuvent d'ailleurs être effectué sur `getNodeNames()`